index.html

```diff
@@ -74,7 +74,12 @@
     </div>
 
     <header id="welcome">
-      <h1>Welcome to Carrigan's Community! 🔥</h1>
+      <div style="text-align: center; margin-top: 10px;">
+              <h1>Welcome to Carrigan's Community! 🔥</h1>
+      <p>  <a href="" class="button">join community</a></p>
```

Comment:
```suggestion
      <p>  <a href="https://github.com/naomis-novas/community/issues/new?assignees=naomi-lgbt&labels=%E2%9C%89%EF%B8%8F+misc%3A+invite+to+org&template=join_org.yml&title=%5BORG%5D+Invite+me+to+the+organization" class="button">join community</a></p>
```

Here's the invite link.